src/SonarQube.Client/Messages/QualityProfileRequest.cs

```diff
@@ -24,8 +24,11 @@ namespace SonarQube.Client.Messages
 {
     public class QualityProfileRequest
     {
+        // TODO: since 6.5 this parameter is named "project"; we should change a couple of months after release of LTS.
```

Comment:
Could you create a ticket for it? As we discussed, we don't want to change it too soon.

Reply:
I would prefer to rely on integration test, not a ticket, because when the test starts to fail we will be alerted. Old tickets tend to be forgotten or deleted... Wdyt?